src/tests/ftest/util/dfuse_utils.py

```diff
@@ -380,6 +381,27 @@ def stop(self):
         # Only assume clean if nothing above failed
         self.__need_cleanup = False
 
+    def get_stats(self):
+        """Return the I/O stats for the filesystem
+
+        Only works if there is one entry in the client list.
+
+        Returns:
+            dict: the json response
```

Comment:
Minor - we often forget this in docstrings
```suggestion
        Raises:
            CommandFailure: if the command fails

        Returns:
            dict: the json response
```